assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -16,6 +61,55 @@ const Cart = () => {
 				</span>
 			</div>
 			<div className="wc-block-cart__sidebar">
+				<h2 className="wc-block-cart__totals-title">
+					{ __( 'Cart totals', 'woo-gutenberg-products-block' ) }
+				</h2>
```

Comment:
Does this follow the correct semantic structure for the cart block on the page (i.e. should it be `h3` or some other heading level)?

Reply:
Is this outlined somewhere? Looking at the designs, I thought _Cart_ would be the `h1` and _Shopping cart_ and _Cart totals_ would `h2`.

Reply:
Oh this was more of a "gut check" comment to make sure it had been considered. From your reply it looks like you have 👍 and h2 is the correct semantics this spot in the rendered tree.